src/scripts/main.js

```diff
@@ -1,3 +1,19 @@
 'use strict';
 
-// write your code here
+const populationData = document.querySelectorAll('.population');
+const populations = [...populationData].map(el => (
+  +el.innerText.split(',').join('')
+));
+
+const total = populations.reduce((prev, curr) => prev + curr, 0);
+const totalData = splitWithCommas(total);
+
+const everage = total / populations.length;
```

Comment:
```suggestion
const average = total / populations.length;
```

Reply:
you can install `spell check` extension: https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker
For me it's very useful 😉